lib/kramdown/converter/html.rb

```diff
@@ -162,24 +170,26 @@ def convert_ul(el, indent)
         elsif !@footnote_location && el.options[:ial] && (el.options[:ial][:refs] || []).include?('footnotes')
           @footnote_location = ZERO_TO_ONETWENTYEIGHT.map { rand(36).to_s(36) }.join
         else
-          format_as_indented_block_html(el.type, el.attr, inner(el, indent), indent)
+          type = el.type == :ul ? "ul" : "ol"
+          format_as_indented_block_html(type, el.attr, inner(el, indent), indent)
```

Comment:
Moving to a string instead of `el.type` is okay if it is a direct exchange. But if we have to check beforehand, even with only two choices, I think that this unnecessary complicates the code. So please remove this part and the others where additional checks are needed.